dcos/http.py

```diff
@@ -44,9 +44,9 @@ def _verify_ssl(verify=None):
 
     if verify is None:
         verify = config.get_config_val("core.ssl_verify")
-        if verify.lower() == "true":
+        if verify and verify.lower() == "true":
```

Comment:
If you believe the pydoc for `get_config_val` then at this point verify should be a `str` or `None`. In that case `verify` is true if `verify is not None and verify != ""`.